src/main/webapp/test/tableSort.jsp

```diff
@@ -3,176 +3,180 @@
-                        <td >-0.1</td>
-                        <td >-10%</td>
-                    </tr>
-
-                    <tr>
-                        <td >10.3</td>
-                        <td >Zhang HaoQiang</td>
-                        <td >04 May 2010</td>
-                        <td >E 0%</td>
-                        <td >-0.4</td>
-                        <td >+99%</td>
-                    </tr>
-
-                    <tr>
-                        <td >-2</td>
-                        <td >Le Minh Khue</td>
-                        <td >14 May 2011</td>
-                        <td >E +20%</td>
-                        <td >-0.5</td>
-                        <td >-20%</td>
-                    </tr>
-
-                    <tr>
-                        <td >-13.5</td>
-                        <td >Chong Kok Wei</td>
-                        <td >05 March 2012</td>
-                        <td >E 0%</td>
-                        <td >-1</td>
-                        <td >N/A</td>
-                    </tr>
-
-                    <tr>
-                        <td >10.35</td>
-                        <td >Tan Guo Wei</td>
-                        <td >21 August 2010</td>
-                        <td >E +20%</td>
-                        <td >-1.51</td>
-                        <td >-99%</td>
-                    </tr>
-
-                    <tr>
-                        <td >10.01</td>
-                        <td >Le Minh Khue</td>
-                        <td >06 April 2011</td>
-                        <td >E +5%</td>
-                        <td >-1.667</td>
-                        <td >N/A</td>
-                    </tr>
-                </tbody>
-            </table>
-            <br> <br> <br>
-        </div>
+  <div class = "container">
+      <table class="table table-striped">
+        <thead>
+          <tr>
+            <th id="button_sortid"
+                class="button-sort-ascending toggle-sort">ID <span class="icon-sort unsorted"></span></th>
+            <th id="button_sortname"
+                class="button-sort-none toggle-sort">Name <span class="icon-sort unsorted"></span></th>
+            <th id="button_sortdate"
+                class="button-sort-none toggle-sort">Date <span class="icon-sort unsorted"></span></th>
+            <th id="button_sortPoint" data-toggle-sort-comparator="sortByPoints"
+                class="button-sort-none toggle-sort">Point <span class="icon-sort unsorted"></span></th>
+            <th id="button_sortPointNumber" data-toggle-sort-comparator="sortByPoints"
+                class="button-sort-none toggle-sort">Point (Number) <span class="icon-sort unsorted"></span></th>
+            <th id="button_sortDiff" data-toggle-sort-comparator="sortByDiff"
+                class="button-sort-none toggle-sort">Diff <span class="icon-sort unsorted"></span></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td >0</td>
```

Comment:
Might as well take this opportunity to fix the extraneous space after `<td` before `>`.